src/jetpack-editor-setup.js

```diff
@@ -129,6 +128,14 @@ export function registerJetpackEmbedVariations( { capabilities } ) {
 	} );
 }
 
+export function enableVideoPressV5Support( { capabilities } ) {
+	if ( ! isActive() || ! capabilities.videoPressV5Support ) {
+		return;
+	}
+
+	require( '../jetpack/projects/plugins/jetpack/extensions/blocks/videopress/editor' );
+}
```

Comment:
As a side note, it would be great if we could cover this logic in a test case [here](https://github.com/wordpress-mobile/gutenberg-mobile/blob/e4abec201fc3d79d32d28752e2477413be787659/src/test/jetpack-editor-setup.js).

Reply:
I agree, I'll add this to [my list](https://github.com/Automattic/jetpack/pull/35637#issuecomment-1961658562) to follow up on next week.